### PR TITLE
fix baichuan2 preprocess scripts

### PR DESCRIPTION
**Pull request by ygdl0228**

### CODE REVIEW

File: examples/dpo/baichuan2/dpo_preprocess_baichuan_parallel.py [hunk 1]
```diff
@@ -66,18 +66,19 @@ def get_logps(model, input_ids, labels, attention_mask, loss_mask):
...
         input_ids = ms.ops.unsqueeze(input_ids, 0)
-    input_ids = P.StridedSlice()(input_ids, (0, 0), (input_ids.shape[0], min(batch_length, input_ids.shape[1] - 1)), (1, 1))
+    input_ids = P.StridedSlice()(input_ids, (0, 0), (input_ids.shape[0], min(batch_length, input_ids.shape[1] - 1)),
+                                 (1, 1))
     labels = ms.Tensor(labels, dtype=ms.int32)
     if len(labels.shape) == 1:
         labels = ms.ops.unsqueeze(labels, 0)
-    labels = P.StridedSlice()(labels, (0, 0), (labels.shape[0], min(batch_length, labels.shape[1] - 1)), (1, 1))
+    labels = P.StridedSlice()(labels, (0, 1), (labels.shape[0], min(batch_length, labels.shape[1])), (1, 1))
     # attention_mask = ms.Tensor(attention_mask, dtype=ms.int32)
```
Comment (ChessQian) [inline examples/dpo/baichuan2/dpo_preprocess_baichuan_parallel.py:75]: 注释代码去掉

Comment (ygdl0228) [reply examples/dpo/baichuan2/dpo_preprocess_baichuan_parallel.py]: 已修改

**Comment by ChessQian** [comment]
fix baichuan2 preprocess scripts

**Comment by ChessQian** [comment]
fix baichuan2 preprocess scripts

**Pull request merged**